src/main/java/com/devonfw/quarkus/ordermanagement/service/v1/model/OrderSearchCriteriaDto.java

```diff
@@ -0,0 +1,21 @@
+package com.devonfw.quarkus.ordermanagement.service.v1.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.ws.rs.DefaultValue;
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+public class OrderSearchCriteriaDto {
+    @DefaultValue("0")
+    private int pageNumber = 0;
+
+    @DefaultValue("10")
+    private int pageSize = 10;
```

Comment:
We just had the same trouble in our other reference app about paging.
We need to align our paging approach and show a homgenous soltuion in both quarkus sample apps.
Please rework and align.

Reply:
Did some rework on it.
Moved the paging stuff from OrderSearchCriteriaDto to an abstract base class [ApplicationSearchCriteriaDto](https://github.com/lilliCao/devon4quarkus-order/blob/create-devon4quarkus-order-application/src/main/java/com/devonfw/quarkus/general/domain/model/ApplicationSearchCriteriaDto.java). Created [PageableJsonDeserializer](https://github.com/lilliCao/devon4quarkus-order/blob/create-devon4quarkus-order-application/src/main/java/com/devonfw/quarkus/general/serialization/PageableJsonDeserializer.java), so that Pageable can be used in the base class. Added [QueryUtil](https://github.com/lilliCao/devon4quarkus-order/blob/create-devon4quarkus-order-application/src/main/java/com/devonfw/quarkus/general/repo/QueryUtil.java) to build the Page (like in the other reference application).
@hohwille is this what you had in mind?